examples/contract/contract_balance_query.py

```
@@ -0,0 +1,103 @@
+
+    receipt = (
+        ContractCreateTransaction()
+        .set_bytecode(bytecode)
+        .set_gas(2_000_000)
+        .set_initial_balance(initial_balance_tinybars)
+        .set_contract_memo("Contract for balance query example")
+        .execute(client)
+    )
+
+    if receipt.contract_id is None:
+        raise RuntimeError("ContractCreateTransaction receipt did not return contract_id")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)